core/templates/dev/head/pages/review-test-page/review-test-page.mainpage.html

```diff
@@ -0,0 +1,83 @@
+{% extends 'dist/base.html' %}
+
+{% block maintitle %}
+  Review Test: {{ story_name }} - Oppia
+{% endblock maintitle %}
+
+{% block header_js %}
+  {{ super() }}
+  <script type="text/javascript">
+    GLOBALS.INTERACTION_SPECS = JSON.parse('{{INTERACTION_SPECS|js_string}}');
+  </script>
+
+{% endblock header_js %}
+
+{% block navbar_breadcrumb %}
+  <div class="nav navbar-nav oppia-navbar-breadcrumb ng-cloak">
+    <span class="oppia-navbar-breadcrumb-separator"></span>
+    <span>
+      Review Test
+    </span>
+  </div>
+{% endblock navbar_breadcrumb %}
+
+{% block content %}
+  <background-banner></background-banner>
+  <div ng-controller="ReviewTest" class="oppia-review_test-container" style="position: relative;">
+    <div class="oppia-review-test">
+      Review Test
+    </div>
+    <question-player ng-if="questionPlayerConfig.skillList &&
+                            questionPlayerConfig.skillList.length > 0"
+                     player-config="questionPlayerConfig">
+    </question-player>
+  </div>
+  <style>
+    .oppia-review_test-container .oppia-review-test {
+      color: #01645c;
+      font-family: 'Capriola', 'Roboto', Arial, sans-serif;
+      font-size: 3em;
+      margin-top: 0px;
+      margin-bottom: 25px;
+      padding-top: 66.5px;
+      text-align: center;
+    }
+    .oppia-review_test-container {
+      color: black;
+      font-family: 'Roboto', Arial, sans-serif;
+      margin: 0 auto;
+      text-decoration: none;
+    }
+
+    @media(max-width: 925px) {
+      .oppia-review_test-container {
+        max-width: 95%;
+      }
+    }
+  </style>
+
+{% endblock %}
+
+{% block footer_js %}
+ {{ super() }}
+  <script src="/third_party/static/ckeditor-4.9.2/ckeditor.js"></script>
+
+  <script src="/templates/dev/head/mathjaxConfig.js"></script>
+  <script src="/third_party/static/MathJax-2.6.0/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
+
+  <script src="/extensions/objects/templates/MathLatexStringEditorDirective.js"></script>
```

Comment:
@vojtechjelinek @Jamesjay4199  just to check, is this correct or should it be replaced with a webpack construct?

Also @sophiewu6 just wondering, why are we including only these three templates and nothing else?

Reply:
This should be replaced with a webpack construct.

Reply:
Hi @vojtechjelinek Could you guide me on how to replace it with a webpack construct? I'm not familiar with it. Thanks!

Reply:
Just add `require('objects/objectComponentsRequiresForPlayers.ts');` to review-test-page.controller.ts

Reply:
To answer your other question @seanlip , I believe this was before webpack where we needed to have dependencies required by a directive within the page that calls the directive. The exploration player related stuff was moved but this was probably remaining.

Reply:
Done.